main/res/layout/brouter_threshold_preference.xml

```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would suggest to separate non-related changes to at least different commit

Reply:
They have been during development, but I normally squash them before finalizing a PR.